docs/gettingStarted/setupGenericScheme.md

```diff
@@ -0,0 +1,130 @@
+# Setup Generic Scheme for a DAO
+
+## When to use Generic Scheme
+[Generic Scheme](https://github.com/daostack/arc/blob/master/contracts/schemes/GenericScheme.sol) allows a DAO to interact with other contracts and execute any method from the contract if the DAO decides to
```

Comment:
please take a look on latest arc (v24) 
Universal GenericScheme is now called UGenericScheme.
see https://github.com/daostack/arc/blob/master/contracts/universalSchemes/UGenericScheme.sol
There is also a none universal GenericScheme  
https://github.com/daostack/arc/blob/master/contracts/schemes/GenericScheme.sol